### PR TITLE
Logzio pump

### PR DESCRIPTION
**Pull request by idohalevi**
Add Logz.io as a pump

### CODE REVIEW

File: pumps/logzio.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	defaultLogzioURL            = "https://listener.logz.io:8071"
+
+	minDiskThreshold = 0
+	maxDiskThreshold = 100
+)
+
+type LogzioPumpConfig struct {
+	CheckDiskSpace bool   `mapstructure:"check_disk_space"`
+	DiskThreshold  int    `mapstructure:"disk_threshold"`
+	DrainDuration  string `mapstructure:"darin_duration"`
```
Comment (buger) [inline pumps/logzio.go]: typo in mapstructure definition

**Comment by buger** [comment]
Looks solid!

**Comment by idohalevi** [comment]
Thank you :)
I fixed the typos

**Comment by idohalevi** [comment]
@buger 
Any idea when you'll merge this pr?

**Comment by buger** [comment]
Should be part of next Pump release. Thank you!

**Comment by idohalevi** [comment]
@buger thanks :)

**Pull request merged**